pkgs/development/python-modules/ofxtools/default.nix

```diff
@@ -21,11 +21,11 @@ buildPythonPackage rec {
     hash = "sha256-NsImnD+erhpakQnl1neuHfSKiV6ipNBMPGKMDM0gwWc=";
   };
 
-  nativeCheckInputs = [ nose ];
+  nativeCheckInputs = [ pytestCheckHook ];
   # override $HOME directory:
   #   error: [Errno 13] Permission denied: '/homeless-shelter'
   checkPhase = ''
-    HOME=$TMPDIR nosetests tests/*.py
+    HOME=$TMPDIR pytest tests/*.py
   '';
```

Comment:
`preCheck` should suffice for this.

Reply:
Oh yeah, I tend to avoid moving the derivation too much, but this can be applied